backend/routes/postReaction.py

```
@@ -264,4 +265,26 @@ def remove_rsvp(event_id: int, db: Session = Depends(get_db), current_user: User
     db.commit()
-    return {"message": "RSVP removed successfully"}
\ No newline at end of file
+    return {"message": "RSVP removed successfully"}
+
+@router.get("/event/{event_id}/my_rsvp/")
+def get_user_rsvp_status(event_id: int, db: Session = Depends(get_db),current_user: User =Depends(get_current_user)):
+    rsvp = db.query(EventAttendee).filter(EventAttendee.event_id == event_id, EventAttendee.user_id == current_user.id).first()
+    return {"status": rsvp.status if rsvp else None}
+
+@router.get("/posts/events/rsvp/counts/")
+def get_rsvp_counts(event_id: int = Query(...), db: Session = Depends(get_db)):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_rsvp_counts`

(ANN201)